app/controllers/api/v1/members_controller.rb

```diff
@@ -1,5 +1,8 @@
 class Api::V1::MembersController < Api::V1::ApiController
-
+	skip_before_action :authenticate_member!, only: [:create]
+  def create
+    WebsiteMember.where( invitation_token: params[:website_member][:token] ).update_all( member_id: Member.find_by_email(params[:website_member][:email]).id, website_id: params[:website_member][:website_id] )
```

Comment:
Ok `update_all` dar de ce setezi website_id la toate??? E in contradictie cu ce ai facut initial. Adica faci update_all ca sa setezi member_id la toate invitatiile posibile trimise pentru toate siteurile. Dar nu trebuie sa setezi website_id sa fie acelasi la toate nici nu ar trebuii sa mearga asta ar trebuii sa ai eroare. Trebuie sa ai un index sau o validare de unicitate pentru member_id si website_id...